common/src/main/java/com/linecorp/centraldogma/internal/jsonpatch/AddOperation.java

```diff
@@ -86,10 +89,16 @@ public AddOperation(@JsonProperty("path") final JsonPointer path,
     }
 
     @Override
-    JsonNode apply(final JsonNode node) {
+    JsonNode apply(JsonNode node) {
         if (path.toString().isEmpty()) {
             return valueCopy();
         }
+        if (node.isNull() && Strings.isNullOrEmpty(path.head().getMatchingProperty()) &&
```

Comment:
JSON patch specification does not consider if the target node is absent.
Let me revert this change since the code is no longer used because we changed the format of mirroring and credentials.